### PR TITLE
making the delete button

### PR DESCRIPTION
**Pull request by gulshan81**
Targets #2190

### Description
making the delete button

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Prospect/Database/Migrations/2020_04_30_174905_create_prospect_checklist_statuses_table.php [hunk 1]
```diff
@@ -23,9 +23,9 @@ public function up()
         });
 
         Schema::table('prospect_checklist_statuses', function (Blueprint $table) {
-            $table->foreign('module_checklist_id')->references('id')->on('module_checklists');
-            $table->foreign('module_checklist_task_id')->references('id')->on('module_checklist_tasks');
-            $table->foreign('prospect_id')->references('id')->on('prospects');
+            $table->foreign('module_checklist_id')->references('id')->on('module_checklists')->onDelete('cascade');
+            $table->foreign('module_checklist_task_id')->references('id')->on('module_checklist_tasks')->onDelete('cascade');
+            $table->foreign('prospect_id')->references('id')->on('prospects')->onDelete('cascade');
```
Comment (gautam071) [inline Modules/Prospect/Database/Migrations/2020_04_30_174905_create_prospect_checklist_statuses_table.php:28]: Can we create a new migration for this?

File: Modules/Prospect/Database/Migrations/2020_04_30_182928_create_prospect_nda_meta_table.php [hunk 2]
```diff
@@ -21,8 +21,8 @@ public function up()
         });
 
         Schema::table('prospect_nda_meta', function (Blueprint $table) {
-            $table->foreign('prospect_id')->references('id')->on('prospects');
-            $table->foreign('nda_meta_id')->references('id')->on('nda_meta');
+            $table->foreign('prospect_id')->references('id')->on('prospects')->onDelete('cascade');
+            $table->foreign('nda_meta_id')->references('id')->on('nda_meta')->onDelete('cascade');
```
Comment (gautam071) [inline Modules/Prospect/Database/Migrations/2020_04_30_182928_create_prospect_nda_meta_table.php:25]: Can we create a new migration for this?

File: Modules/Prospect/Database/Migrations/2020_05_02_113031_create_prospect_requirements_table.php [hunk 3]
```diff
@@ -25,7 +25,7 @@ public function up()
         });
 
         Schema::table('prospect_requirements', function (Blueprint $table) {
-            $table->foreign('prospect_id')->references('id')->on('prospects');
+            $table->foreign('prospect_id')->references('id')->on('prospects')->onDelete('cascade');
```
Comment (gautam071) [inline Modules/Prospect/Database/Migrations/2020_05_02_113031_create_prospect_requirements_table.php:28]: Can we create a new migration for this?

File: Modules/Prospect/Database/Migrations/2020_05_04_150517_create_prospect_calendar_meeting_table.php [hunk 4]
```diff
@@ -21,8 +21,8 @@ public function up()
         });
 
         Schema::table('prospect_calendar_meeting', function (Blueprint $table) {
-            $table->foreign('prospect_id')->references('id')->on('prospects');
-            $table->foreign('calendar_meeting_id')->references('id')->on('calendar_meetings');
+            $table->foreign('prospect_id')->references('id')->on('prospects')->onDelete('cascade');
+            $table->foreign('calendar_meeting_id')->references('id')->on('calendar_meetings')->onDelete('cascade');
         });
```
Comment (gautam071) [inline Modules/Prospect/Database/Migrations/2020_05_04_150517_create_prospect_calendar_meeting_table.php:26]: Can we create a new migration for this?

File: Modules/Prospect/Resources/views/index.blade.php [hunk 5]
```diff
@@ -34,11 +35,15 @@
...
                           <a href="{{ route('prospect.show', $prospect) }}">{{ $prospect->name }}</a>  
                         </td>
+                        <td>{{$prospect->Resources}}</td>
+                        <td>{{$prospect->Requirement}}</td>
                         <td>
-                            
-                        </td>
-                        <td>
-                            
+                            {{-- <a class="btn btn-danger" href="{{ route('prospect.destroy', $prospect->id)}}">Delete</a> --}}
```
Comment (gautam071) [inline Modules/Prospect/Resources/views/index.blade.php:41]: Remove the unwanted code

**Review by gautam071 — CHANGES_REQUESTED** [review]
Left some comments

File: Modules/Prospect/Resources/views/index.blade.php (hunk 5 above)
Comment (gulshan81) [reply Modules/Prospect/Resources/views/index.blade.php]: I removed the unwanted code.

**Comment by gautam071** [comment]
@gulshan81 Sorry for changing the requirement at the last minute 
Can we implement the soft delete

**Pull request merged**